Spec/PushActivationStateMachine.swift

```diff
@@ -293,7 +293,14 @@ class PushActivationStateMachine : QuickSpec {
                         guard let rawBody = request.httpBody else {
                             fail("should have a body"); return
                         }
-                        guard let body = stateMachine.rest.defaultEncoder.decode(rawBody, error: nil) as? NSDictionary else {
+                        let decodedBody: Any
```

Comment:
This started failing because I changed the `- (id)decode:(NSData *)data error:(NSError *_Nullable *_Nullable)error;` header to the right form for Swift interoperability (`- (nullable id)decode:(NSData *)data error:(NSError *_Nullable *_Nullable)error;`). The `error` argument should be translated to a a `@throw` (Swift Error Handling).